UNITTESTS/empty_baseline/unittest.cmake

```diff
@@ -6,7 +6,6 @@
 set(unittest-includes ${unittest-includes}
   .
   ..
-  ../features/mbedtls/mbed-crypto/inc/mbedtls/
```

Comment:
instead of removal, you might want to change this to `../features/mbedtls/inc/mbedtls/` ?
because that is where the new files located?

Reply:
Good point, this should be fixed now.